packages/site/src/ui/wallet/containers/WalletConsole/WalletConsole.tsx

```
@@ -0,0 +1,194 @@
+  const handleSendSubmit = (params: SendParams) => {
+    const sendParams: SendParams = { ...params };
+    // Only include destinationTag if it's a valid integer
+    if (params.destinationTag !== undefined) {
+      const tag = Number(params.destinationTag);
+      if (!isNaN(tag) && Number.isInteger(tag) && tag >= 0) {
+        sendParams.destinationTag = tag;
+      }
+    }
+    // Remove destinationTag if it's not a valid non-negative integer
+    if (typeof sendParams.destinationTag !== 'number' || !Number.isInteger(sendParams.destinationTag) || sendParams.destinationTag < 0) {
+      delete sendParams.destinationTag;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)